openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/minimal/build.sbt

```diff
@@ -1,13 +1,13 @@
 lazy val root = (project in file("."))
```

Comment:
**question:** There is no `test` in this `minimal` project. How is it used?

Reply:
Ah ok, I thought `minimal` was added by you, but it was already there, with its proper `test`, thanks!